app/core/formatters.js

```diff
@@ -24,6 +24,15 @@ export const formatToRoundedShortNumber = (value: number): string => {
     .toString()
 }
 
+export const convertToArbitraryDecimals = (
+  num: number,
+  decimals: number = 8,
+): string => {
+  // eslint-disable-next-line
+  const multiplier = 1 / Math.pow(10, decimals)
+  return (num * multiplier).toFixed(decimals)
```

Comment:
I'd suggest using the bignumber.js package (already part of this repo) for this map to avoid any loss of precision.  There are some helpful functions in [math.js](https://github.com/CityOfZion/neon-wallet/blob/aef2ec4d59a6f7f1590d30a94974b6af3d92e982/app/core/math.js) that could be used here.

```js
export const convertToArbitraryDecimals = (
  num: number,
  decimals: number = 8,
): string => {
  return toBigNumber(num).times(multiplier).div(Math.pow(10, decimals)).toFixed(decimals)
}
```